### PR TITLE
WIP: update upcast

### PR DESCRIPTION
**Pull request by phunehehe**

### CODE REVIEW

File: pkgs/default.nix [hunk 1]
```diff
@@ -310,8 +310,8 @@ in rec {
 
   thumbor = (import ./thumbor { inherit pkgs newrelic-python statsd; }).thumbor;
 
-  upcast = pkgs.haskellPackages.callPackage ./upcast {
-    inherit awsEc2 vkPosixPty vkAwsRoute53;
+  upcast = pkgs.haskellngPackages.callPackage ./upcast {
+    inherit vk-posix-pty;
```
Comment (phunehehe) [inline pkgs/default.nix:314]: is there a reason to keep vk-posix-pty here though? it's already in nixpkgs-14.12

**Comment by phunehehe** [comment]
Doesn't seem to work with nixpkgs-14.12 though, finding stuff to backport...

File: pkgs/default.nix (hunk 1 above)
Comment (proger) [inline pkgs/default.nix:314]: i have no idea who maintains it there

**Comment by proger** [comment]
i wonder if it's possible to use haskell stuff from master (or whatever branch you're using) and our old stdenv. this would let you backport the whole haskell thing at once.

**Comment by phunehehe** [comment]
obsoleted

**Pull request closed without merging**